README.md

```diff
@@ -1,5 +1,7 @@
 # SlicerWMA
 
+Note: this extension is being deprecated in favor of the new and improved [SlicerTractParcellation](https://github.com/SlicerDMRI/SlicerTractParcellation)!
```

Comment:
```suggestion
> [!IMPORTANT]  
> This extension is being deprecated in favor of the new and improved [SlicerTractParcellation](https://github.com/SlicerDMRI/SlicerTractParcellation) extension.
```

Reply:
Here is a preview of the suggested changes:

![image](https://github.com/SlicerDMRI/SlicerWMA/assets/219043/97143e05-ff21-4523-8082-ae6a9404b637)